README.adoc

```diff
@@ -1,11 +1,22 @@
-= Address Book (Level 3)
+= Organice (Organ Transplant Manager)
```

Comment:
Are we stylising the application name to Oraganice or ORGANice?

Reply:
ORGANice. Thanks for pointing, did the changes already